moveit_core/robot_model/src/planar_joint_model.cpp

```diff
@@ -146,11 +146,23 @@ void computeTurnDriveTurnGeometry(const double* from, const double* to, double&
 {
   dx = to[0] - from[0];
   dy = to[1] - from[1];
-  drive_angle = atan2(dy, dx);
-
-  initial_turn = angles::shortest_angular_distance(from[2], drive_angle);
+  const double angle_straight_diff =
+      std::hypot(dx, dy) > 0.01 ? angles::shortest_angular_distance(from[2], std::atan2(dy, dx)) : 0.0;
```

Comment:
0.01 seems nowhere near float precision. Would something bad happen if you made it (a small multiple of) `std::numeric_limits<float>::min()`?

Reply:
What do you think about adding it as a joint property .? I'm pretty sure this will depend on the robot, maybe some robots are ok with having some extra rotational movement to achieve a very small translational movement 

We call it something like
```bash
min_translation_distance # If the translational distance between two states is less than min_translation_distance the movement will be pure rotation
```

Reply:
Sounds good to me.